packages/cache-inmemory/package.json

```diff
@@ -26,7 +26,7 @@
   },
   "scripts": {
     "watch": "tsc --build --watch",
-    "prepare": "cross-env NODE_ENV=production yarn run build",
```

Comment:
question: shouldn't we keep the prepare script for the publish command to execute it and the prepare would call the prepublishOnly?

Reply:
Why do you want to keep both?

Reply:
because the publish command wont call the build, so I am not sure if the build will be available right?